content/docs/contributing/building.md

```diff
@@ -3,123 +3,211 @@ title: Building cert-manager
 description: 'cert-manager contributing guide: Building cert-manager'
 ---
 
-cert-manager makes use of [Bazel](https://bazel.build/) to build the project. 
-Bazel manages all developer dependencies, Helm chart building, Docker images and the code itself. 
-We try to use it as much as possible.
-We currently use Bazel `v4.2.1`. The minimum supported version is `v4.0.0`.
+cert-manager is built and tested using [make](https://www.gnu.org/software/make/manual/make.html), with a focus on using the standard Go tooling
```

Comment:
Question: does it matter what version of make we use (i.e in CI)? Is it worth mentioning it here?

Reply:
It _basically_ doesn't matter - I've lightly tested our build system going as far back as CentOS 7 (released in 2014) and it works fine with Make 3.82 there. I'll add 3.82 as a minimum, but realistically everyone running Linux or macOS today who's doing K8s development will have a new enough version for everything to work just fine!